src/DataProvider/Pagination.php

```diff
@@ -91,6 +91,10 @@ public function getOffset(string $resourceClass = null, string $operationName =
             return ($offset = ($context['count'] ?? 0) - $last) < 0 ? 0 : $offset;
         }
 
+        if (!\is_int(($this->getPage($context) - 1) * $limit)) {
```

Comment:
Could you add a variable to avoid duplication?

Reply:
Done